ovs_performance.py

```diff
@@ -272,10 +275,30 @@ def test_p2v2p(nr_of_flows, packet_sizes):
 
             tester.start_traffic(config.tester_interface)
 
-            warm_up_verify(nr_of_flows * 2, DEFAULT_WARM_UP_TIMEOUT)
-
+            warm_up_error = warm_up_verify(nr_of_flows * 2,
+                                        config.warm_up_timeout)
             tester.stop_traffic(config.tester_interface)
 
+            if warm_up_error:
+                if config.warm_up_no_fail:
+
+                    stop_traffic_loop_on_vm(config.dut_vm_address)
+
+		    lprint("INFO: Waiting for Datapath flows to flush")
```

Comment:
I think this can be removed, as there will be "lprint("  * Doing flow table cool-down...")" in the flow_table_cool_down() function